### PR TITLE
[FEATURE] Add text that is displayed upon entering to a space

### PR DESCRIPTION
**Pull request by SandrinePDR**
Initial version. Entry text can be defined and is displayed, when a character enters a space

### CODE REVIEW

File: pandora-client-web/src/ui/screens/spaceConfiguration/spaceConfiguration.tsx
Comment (Sekkmer) [inline pandora-client-web/src/ui/screens/spaceConfiguration/spaceConfiguration.tsx:70]: why is it called `fluffText` I wouldn't be able to guess what it is used for unless I also find the label for the user input, maybe it could be renamed to something more intuitive like:

`welcomeText`

Comment (SandrinePDR) [reply pandora-client-web/src/ui/screens/spaceConfiguration/spaceConfiguration.tsx:70]: Oh that's just me. "Fluff text" is text used on board games to fluff out game components without addin new mechanics or rules. But I understand that it is confusing and will change it.

**Review by Sekkmer — APPROVED** [review]
looks good for me^^

(as per review rules please wait for a second review or until 3 days has passed since the creation of this PR which should be tomorrow before merging)

**Review by Jomshir98 — CHANGES_REQUESTED** [review]
Looks and works mostly good. There are, however some problems; noted all of them below.

File: pandora-client-web/src/ui/screens/spaceConfiguration/spaceConfiguration.tsx
Comment (SandrinePDR) [reply pandora-client-web/src/ui/screens/spaceConfiguration/spaceConfiguration.tsx]: I changed the size of the description from 16 to 8 to maintain the size of the dialog. Should we rather decrease the other size as well or risk a scrollbar for the space creation dialog?
Also: wouldn't it be a good idea to use a constant for this value?

**Review by Jomshir98 — APPROVED** [review]
Looks good to me!

**Pull request merged**
